src/features/moviesPage/moviePage/BigMovieTile/styled.js

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+
+export const BackGround = styled.div`
+    background-color: ${({theme}) => theme.color.black};
+    color: ${({theme}) => theme.color.white};
+`;
+
+export const Backdrop = styled.img`
+    display: block;
+    width: 100%;
+`;
+
+export const BackDropWrapper = styled.div`
+    max-width: 1368px;
+    position: relative;
+    margin: auto;
+`;
+
+export const Information = styled.div`
+    background: ${({theme}) => theme.color.lineargradient};
+    background-position: center;
+    background-size: 140%;
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: bottom;
+`;
+
+export const Title = styled.h1`
```

Comment:
here it is necessary to change this h1 tag. The title of the entire page in the header is "movie browser" and it already has an h1 tag- btw this element must be replaced with a ready-made PageHeader component anyway.